jboss/container/openjdk/jdk/11/configure.sh

```diff
@@ -13,20 +13,12 @@ pushd ${ARTIFACTS_DIR}
 cp -pr * /
 popd
 
-# As of rhel 7.6, rh-maven35 pulls in jdk8, so we need to remove them
```

Comment:
This is a bug in the maven SCL package. Has it been reported in Bugzilla?

Reply:
> Yes, but it won't be fixed. The dependency for rh-maven35 is on java-devel and jdk 11 provides java-11-devel,

I think jdk 11 should be providing both `java-devel` and `java-11-devel`.

Reply:
> I think jdk 11 should be providing both java-devel and java-11-devel.

Sorry for the terseness of my message. To expand: I'm interested in pursuing fixing this in the right plaec (rpm permissions)) with my new team hat on (or at least learning why it will never be fixed). But in the mean time, working around it in the module as with this PR is the right thing to do (as we did for the JDK7→8 transition period, when the same thing happened). So I will review this PR properly ASAP